src/lib/builders/pin-input/types.ts

```diff
@@ -55,6 +55,13 @@ export type CreatePinInputProps = {
 	 * @see https://melt-ui.com/docs/controlled#change-functions
 	 */
 	onValueChange?: ChangeFn<string[]>;
+
+	/**
+	 * Whether you want to hide the input with a mask.
+	 *
+	 * @default false
+	 */
+	mask?: boolean;
```

Comment:
Rather than a boolean, maybe it should allow a string? So they can choose what appears in the input.

Reply:
Yes, that's definitely better than boolean, allows for more flexibility. I'll update it.